src/Aspire.Cli/api/aspire.cs

```diff
@@ -0,0 +1,15 @@
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+namespace Aspire.Cli
+{
+    public partial class Program
```

Comment:
@mitchdenny @maddymontaquila @davidfowl separate from the fact that they shouldn't be showing up here (which I'll fix) This package is today set to be marked as stable for 9.2. Is that intentional or should we keep it as preview?

Reply:
We need to decide.

Reply:
I agree, I'll put up a PR.

Reply:
https://github.com/dotnet/aspire/pull/8387